base/src/ast.rs

```diff
@@ -274,12 +272,17 @@ pub struct ValueBinding<Id: AstId> {
 /// call `walk_mut_*` to continue traversing the tree.
 pub trait MutVisitor {
     type T: AstId;
+
     fn visit_expr(&mut self, e: &mut SpannedExpr<Self::T>) {
         walk_mut_expr(self, e);
     }
+
     fn visit_pattern(&mut self, e: &mut SpannedPattern<Self::T>) {
         walk_mut_pattern(self, &mut e.value);
     }
+
+    fn visit_typ(&mut self, _: &mut AstType<<Self::T as AstId>::Untyped>) {}
```

Comment:
Fair enough. As an aside, I use `typ` in part because its `type` in swedish so it seemed rather fitting. Its also nice that it is one letter more than `ty` making it stand out a bit more.